### DISCUSSION TITLE
not working

### DISCUSSION THREAD

**Discussion by krstan4o** [merlinschumacher/Open-Covid-Certificate-Validator#8] [Q&A] [answered]
Each cert is invalid

**Comment by MFISCHER-DEV-LAB** [comment]
I can relate to the same issue. When using the Docker version and calling the API, no certificate is marked as valid although the basic demo shows it as valid.

**Comment by merlinschumacher** [comment, marked as answer]
Did you set the env variable CERT_COUNTRY to DE?

**Comment by Samuel-Dittmann** [comment]
For me I needed to add the cert Country Variable when starting the container. So it looks like this now:
docker run -p 8000:8000 -e CERT_COUNTRY=DE ghcr.io/merlinschumacher/open-covid-certificate-validator:main

**Reply by merlinschumacher** [reply]
Yep thats the way to go.
